types/budget.d.ts

```diff
@@ -53,7 +53,7 @@ declare global {
       }
 
       /** Supported timing metrics. */
-      export type TimingMetric = 'first-contentful-paint' | 'first-cpu-idle' | 'interactive' | 'first-meaningful-paint' | 'max-potential-fid' | 'estimated-input-latency' | 'total-blocking-time' | 'speed-index';
+      export type TimingMetric = 'first-contentful-paint' | 'first-cpu-idle' | 'interactive' | 'first-meaningful-paint' | 'max-potential-fid' | 'estimated-input-latency' | 'total-blocking-time' | 'speed-index' | 'largest-contentful-paint' | 'cumulative-layout-shift';
```

Comment:
any thoughts on the `cumulative-layout-shift` is not a timing problem? :)

Reply:
can we just rename to `metric budget`?

Reply:
kind of a big budgets breaking change, and I believe the steer away from metric in the first place was that tracking your file size over time is basically a "metric" and it's a pretty narrow LH view to say only these things we bless as metrics are "metrics"

I'm not sure budgets.json has taken off in the broader tooling space in the way we once hoped it would though. AFAIK it's basically stayed LH adjacent, so rename may be considered at this point?